src/main/resources/application.yaml

```diff
@@ -30,11 +30,11 @@ spring:
 
   mail:
     from: no-reply@reform.hmcts.net
-    to:  Praveen.Saxena@HMCTS.NET
-    subject:  Exception received at Rout
+    to:  sushant.choudhari@hmcts.net,abhijit.diwan@hmcts.net
```

Comment:
in the production also you will keep same email ids?

Reply:
nope its for testing purpose only. Email address will be replaced after discussion with Praveen and dev-ops team